### DISCUSSION TITLE
Add detailed documentation

### DISCUSSION THREAD

**Discussion by DanickBG** [grafana/grafonnet#184] [Q&A] [answered]
Is is possible to have a separate document with a basic documentation and list of all possible panels and what their syntax looks like in Jsonnet format? 
Thank you!

**Comment by Duologic** [comment, marked as answer]
There are docs on all panels here: https://grafana.github.io/grafonnet/API/panel/index.html

And examples here: https://grafana.github.io/grafonnet/examples/runtimeDashboard.html

Is that sufficient for you? If not, can you explain where you are stuck or what you don't understand?

**Reply by DanickBG** [reply]
Ah thank you! I somehow missed this :)
